_assets/stylesheets/pages/_locations.scss

```diff
@@ -98,10 +98,68 @@
 
 .launch-form-header {
   color: $cr-blue;
-  
+
   h1 {
     color: $cr-blue;
     font-family: $condensed-extra-font-face !important;
     font-weight: 500 !important;
   }
 }
+
+.location-overlay {
+  background-color: rgba(0, 0, 0, 0.8);
+
+  .modal-dialog {
+    width: 90%;
+    margin: auto;
+
+    @media (min-width: $screen-md) {
+      width: 70%;
+    }
+
+    @media (min-width: $screen-lg) {
+      width: 60%;
+    }
+  }
+}
+
+.location-overlay-content {
+  img {
+    display: block;
+    margin-top: 60px;
```

Comment:
Too much margin. This does not look great on mobile like this. I like it at zero
![Screen Shot 2021-08-13 at 3 35 26 PM](https://user-images.githubusercontent.com/32345656/129409949-d068cb3d-f71b-4e1a-ac0a-c36ebe5cde3e.png)